stellarphot/gui_tools/seeing_profile_functions.py

```diff
@@ -574,6 +574,8 @@ def _update_plots(self):
         if self.rad_prof is None:
             return
 
+        fwhm = 2 * self.rad_prof.HWHM
```

Comment:
So here you are setting the FWHM to 2*HWHM, what did you use previously?  Or I guess, based on line 608 that you are replacing putting the cutoff radius as a vertical line in the plot and making it FWHM?   Might want to clarify this in comments.

Reply:
There are two places below that need the FWHM. It turns out that `rad_prof` has a `FWHM` attribute, so now I just use that in the two places that the FWHM is needed.